hack/release.sh

```diff
@@ -114,6 +114,92 @@ s3_url() {
 	esac
 }
 
+map_goos_to_uname() {
+	# we need to map our GOOS to uname values
+	# see https://en.wikipedia.org/wiki/Uname
+	# ie, GOOS=linux -> "uname -s"=Linux
+	GOOS=$1
+	case "$GOOS" in
```

Comment:
I think it's has clearer meaning of using a named local variable for the argument, but it's fine to use `$1` as it's the only argument of the function. I will change to single line style.